### PR TITLE
Change Focus handlers

### PR DESCRIPTION
**Pull request by sbrack8t**

### CODE REVIEW

File: js/src/primary-menu.es6.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ import Drupal from 'drupal';
 
   Drupal.behaviors.primaryNav = {
     attach(context) {
-      const submenuNode = context.querySelector('.menu--main');
+      const submenuNode = context.querySelector('.menu--submenu');
```
Comment (kmonahan) [inline js/src/primary-menu.es6.js]: I don't think this is correct. The MenuBar should be the horizontal menu with the top-level links, which is menu--main. (Why the variable is called submenuNode is an excellent question. I think we should change that to something more sensible like menuNode.) I'm basing this off of https://www.w3.org/TR/wai-aria-practices/examples/menubar/menubar-1/menubar-1.html, where the top-level menu has the role of 'menubar'.

File: js/src/modules/_Menu.es6.js [hunk 2]
```diff
@@ -175,7 +183,8 @@ export class PopupMenu extends Menu {
   }
 
   setFocusToItem(newItem) {
-    newItem.focus();
+    //eslint-disable-next-line
```
Comment (kmonahan) [inline js/src/modules/_Menu.es6.js]: Can we remove this? What linting problem are you hitting with the next line?

**Review by kmonahan — CHANGES_REQUESTED** [review]
See comment about changing the menu selector. Happy to discuss that one further.

File: js/src/primary-menu.es6.js (hunk 1 above)
Comment (sbrack8t) [reply js/src/primary-menu.es6.js]: That makes sense. I'll update.

File: js/src/modules/_Menu.es6.js (hunk 2 above)
Comment (sbrack8t) [reply js/src/modules/_Menu.es6.js]: I had a debugger statement after the line. Forgot to remove the comment

**Pull request merged**